tests/test_check_year.py

```diff
@@ -0,0 +1,38 @@
+"""
+Test check_year(y,s) 
+
+@author: unlu
+"""
+from ixmp.utils import check_year
+import pytest
+
+"Enter a string value for y to raise a ValueError. - in this case test should be successfull"
+ 
+def test_check_year():
+   
+    y1= "a"
```

Comment:
Ensure you use a linter; there should be a space on either side of the ` = ` assignment operator. See [PEP 8](https://www.python.org/dev/peps/pep-0008/#other-recommendations).

Actually, I'm not sure why Stickler is not running on your PR; I will investigate.